Add tech example: YAML ellipses (#517)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/style_guidelines/code-commands.adoc

If you include a shell prompt to indicate that a user with root privileges must run the command, also include a statement about this requirement in the step text, the introductory text, or the prerequisites.
====
+

> * When a sample command includes `sudo`, use the `$` prompt, not `#`, as shown in the following example:

Reviewer: ```suggestion
* When a sample command includes `sudo`, use the dollar sign (`$`) to denote a regular user. Do not use the number sign (`#`) because this sign is reserved for the root user.
+
.Example command
[source,terminal]
----
```

Author: This is out of scope of my PR.

Reviewer: NP. Thanks!

.Additional resources
* link:https://www.redhat.com/sysadmin/difference-between-sudo-su[Exploring the differences between sudo and su commands in Linux]

> [[ellipses-in-yaml-code-blocks]]

Reviewer 2: Should this section be moved to the **Formatting** section instead of the **Technical examples** section?

Author: I wasn't sure that this was formatting. The topic seemed closer to 'syntax highlighting'.

== Ellipses in YAML code blocks

> Use the pound sign (`#`) to comment out an ellipsis in YAML code blocks.

Reviewer 2: ```suggestion
Use the number sign (`#`) to comment out an ellipsis in YAML code blocks.
```

See the "Punctuation and symbols" section of _IBM Style_. The pound sign is a different symbol.

Author: Strange. I don't see the change yet, although I pushed the commit.

> Use the pound sign (`#`) to comment out an ellipsis in YAML code blocks.

Reviewer: ```suggestion
Use the number sign (`#`) to comment out an ellipsis in YAML code blocks.
```

![Screenshot from 2024-10-07 09-24-31](https://github.com/user-attachments/assets/a934b85a-5a2e-495c-8184-5909d4d96782)

Author: I changed that and the commit did not appear. Will try again.

YAML reserves `...` to indicate the end of a document without starting a new document.

.Example AsciiDoc
[source,yaml]
----
apiVersion: operator.openshift.io/v1alpha1
kind: CertManager
metadata:
  name: cluster
# ...
----

> 

Reviewer 2: It would be helpful to add some text to show how that code is rendered, similar to the example in the [Commands requiring root privileges](https://redhat-documentation.github.io/supplementary-style-guide/#commands-with-root-privileges) section.

Author: My preview shows the code rendering as a YAML code block.
